src/foam/u2/filter/FilterView.js

```diff
@@ -206,10 +206,14 @@ foam.CLASS({
 
         if ( searchColumns ) return searchColumns;
 
-        if ( of.model_.searchColumns ) return of.model_.searchColumns;
-
-        if ( of.model_.tableColumns ) {
-          return of.model_.tableColumns.filter(function(c) {
```

Comment:
changing of.model_. to of.getAxiomByName
as it doesn't work for refinements I think
or at least for UserCapabilityJunctionRefine

Reply:
Your changes are correct.